skimage/restoration/_denoise.py

```diff
@@ -632,9 +632,9 @@ def _wavelet_threshold(image, wavelet, method=None, threshold=None,
     """
     wavelet = pywt.Wavelet(wavelet)
     if not wavelet.orthogonal:
-        warn(("Wavelet thresholding was designed for use with orthogonal "
-              "wavelets. For nonorthogonal wavelets such as {}, results are "
-              "likely to be suboptimal.").format(wavelet.name))
+        warn((f'Wavelet thresholding was designed for use with orthogonal '
+              f'wavelets. For nonorthogonal wavelets such as {wavelet.name}, results are '
+              f'likely to be suboptimal.'))
```

Comment:
Thanks for the feedback this is what was in the original code base they held these strings in a tuple so I used the same object as a result. Looking at the `warn` object from `_warnings.py` in the `_ shared` directory there is no impact in using a string directly. However with the fstrings being in a tuple you don't have to include a back slash at the end of each new line of text to avoid getting an IndentationError. Hence putting it in a tuple is potentially less error prone. Is this the potential reason why the code was written like this originally does any of the OGs know?

Reply:
This is actually already just a string and not a tuple, since there is no comma within the parenthesis. I don't think there is a good reason to keep it this way and would just remove the extra parentheses.

Reply:
Cool thanks for the input @grlee77 and @mkcor I will amend this item then.